testdata/goldens/java/com.google.cloud.speech.v1.LongRunningRecognizeMetadata.Builder.html

```diff
@@ -221,12 +224,10 @@ <h5>Inherited Members</h5>
       <span class="xref">java.lang.Object.wait(long,int)</span>
     </div>
   </div>
-  <h6><strong>Namespace</strong>: </h6>
-  <h6><strong>Assembly</strong>: .dll</h6>
-  <h5 id="com_google_cloud_speech_v1_LongRunningRecognizeMetadata_Builder_syntax">Syntax</h5>
-  <div class="codewrapper">
-    <pre><code class="prettyprint">public static final class LongRunningRecognizeMetadata.Builder extends GeneratedMessageV3.Builder&lt;LongRunningRecognizeMetadata.Builder&gt; implements LongRunningRecognizeMetadataOrBuilder</code></pre>
-  </div>
+  <h2>Namespace</h2>
+  
+  <h2>Assembly</h2>
+  <p>.dll</p>

```

Comment:
Namespace and Assembly should not be included if there isn't a corresponding value.

Reply:
Namespace is updated in #173

Reply:
Updated!